helm/oncall/templates/_env.tpl

```diff
@@ -4,15 +4,18 @@
 - name: SECRET_KEY
   valueFrom:
     secretKeyRef:
-      name: {{ template "oncall.fullname" . }}
+      name: {{ template "snippet.oncall.secret.name" . }}
       key: SECRET_KEY
 - name: MIRAGE_SECRET_KEY
   valueFrom:
     secretKeyRef:
-      name: {{ template "oncall.fullname" . }}
+      name: {{ template "snippet.oncall.secret.name" . }}
       key: MIRAGE_SECRET_KEY
 - name: MIRAGE_CIPHER_IV
-  value: "1234567890abcdef"
+  valueFrom:
```

Comment:
Not sure if we can change this value on the fly as it can break existing installations. Let's keep the default value https://github.com/luojilab/django-mirage-field#mirage_cipher_iv
This field can be considered as non secret

Reply:
Okay. I've restore the default and put warning in the values file.